src/bidiMapper/domains/input/ActionDispatcher.ts

```diff
@@ -822,3 +820,70 @@ function getCdpButton(button: number) {
       return 'none';
   }
 }
+
+function getTilt(
+  action:
+    | {tiltX?: number; tiltY?: number}
+    | {azimuthAngle?: number; altitudeAngle?: number}
+): {tiltX: number; tiltY: number} {
+  let tiltX = 0;
+  let tiltY = 0;
+  if ('tiltX' in action || 'tiltY' in action) {
+    tiltX = action.tiltX ?? tiltX;
+    tiltY = action.tiltY ?? tiltY;
+  }
+  if ('azimuthAngle' in action || 'altitudeAngle' in action) {
+    // https://w3c.github.io/pointerevents/#converting-between-tiltx-tilty-and-altitudeangle-azimuthangle
+    const altitudeAngle = action.altitudeAngle ?? 0;
+    const azimuthAngle = action.azimuthAngle ?? 0;
+    let tiltXRadians = 0;
+    let tiltYRadians = 0;
+
+    if (altitudeAngle === 0) {
+      // the pen is in the X-Y plane
+      if (azimuthAngle === 0 || azimuthAngle === 2 * Math.PI) {
+        // pen is on positive X axis
+        tiltXRadians = Math.PI / 2;
+      }
+      if (azimuthAngle === Math.PI / 2) {
```

Comment:
does this comparison on floats work? I think we need to compare with some precision

Reply:
```
6.283185307179586 === Math.PI * 2 // true
```

Reply:
what about `6.2831853071795867` or `6.2831853071795864`?

Reply:
Interesting though, I guess this works as expected 👍